itests/standalone/extra/src/main/resources/classloading/exported-path-patterns.txt

```diff
@@ -26,6 +26,7 @@
 	^org/apache/xmlbeans/impl/schema
 	^org/apache/xmlbeans/impl/values
 	^org/apache/zookeeper(.*)
+    ^com/braintreegateway(.*)
```

Comment:
Minor thing - please can you move this up with the other `com/*` packages to preserve the alphabetic order